### DISCUSSION TITLE
CHILD_SA Stucks in DOWN State

### DISCUSSION THREAD

**Discussion by shmeleviv** [strongswan/strongswan#2526] [General]
Hello,colleagues!
Recently i observed strange behavior:

1) initially, child_sa works fine
2) after received DELETE from remote side (or planned rekeying, at this moment i'm not completely sure), CHILD_SA stuck in DOWN state and not deleted
3) starting from thiat point, communication over the tunnel was not possible and did not trigger a renegotiation of the CHILD_SA (traffic is initiated from stronswan side)

Could you please:
1) Share bugs\defects if exist
2) Maybe there is somewhere  information about this behavior
3) Suggest tshoot steps.

Thank in advance

**Comment by tobiasbrunner** [comment]
See https://github.com/strongswan/strongswan/discussions/1877#discussioncomment-11213287
